packages/create-sitecore-jss/src/templates/angular-sxp/src/app/components/app-components.shared.module.ts

```diff
@@ -20,6 +20,7 @@ import { JssModule } from '@sitecore-jss/sitecore-jss-angular';
     RouterModule,
     JssModule,
     FormsModule,
+    StyleguideSpecimenComponent
```

Comment:
Is there a breaking change in v20 that requires it to be registered in "imports"?